cfpb/sbl-frontend
# (Review second) Update FI profile - phase3 (part II)

Part 2 of #222 

## Changes
- Populates form's `defaultValues` with existing Institution data
- Implements `Clear form` functionality
- Removes previous method of providing form input defaults
- Removes vague `id=stuff`
- Temp fix for TextArea resizing issue

## TODO
- Lots of Typescript stuff (Will work to address these issues in my next PR)

## How to test this PR
1. In browser console, ensure routing is enabled (`setIsRoutingEnabled(true)`)
1. Login with a user who has an associated institution
2. Click on the institution to `View` it's profile
3. Click on `Update your financial institution profile`
5. Modify a single field
6. Hit the `Clear form` button, observe that the edited data is reset
5. Modify a single field
8. Hit `Submit`
9. Verify you are redirected to the `/summary` route and shown a simulated "Success" confirmation
10. In console, look for `data to be submitted (before format)` and confirm the field you edited reflects the data you entered
